backend/seeds/statuses.js

```diff
@@ -0,0 +1,28 @@
+const { tableName } = require('../src/models/status')
+
+const SEED_USERS = [
```

Comment:
I think should be `SEED_STATUSES`?

Reply:
Yes you are right, thank you. Will update the line in the next commit.

Reply:
Done!